### PR TITLE
Juniper upgrade - allow custom white label command to create sites

### PR DESCRIPTION
**Pull request by johnbaldwin**
Reason for this commit is the PR:

https://github.com/appsembler/edx-platform/pull/875

A custom edx-platform command now exists to create sites and configurations and add these to a new `TrustedApplication` model which replaces the old `TrustedClient` model that was in Hawthorn.

The reason for the custom command is to reduce customization of edx-platform core code

The default behavior is to use the existing `create_sites_and_configurations` Django management command that we can override in `--extra-vars` (edx-configs)

If we do not call the `whitelabel` role when running a deploy, then there's no point to this PR and we have to go back and trace through how we mark OAuth applications as trusted so as to not wipe our app tokens (like ecommerce, discovery, AMC) when there's a password reset

### CODE REVIEW

**Review by OmarIthawi — APPROVED** [review]

File: playbooks/roles/whitelabel/defaults/main.yml [hunk 1]
```diff
@@ -4,3 +4,6 @@ WHITELABEL_COURSE_TITLE: "WL_E2E-Test"
 WHITELABEL_COURSE_NUM: "WL_E2E"
 WHITELABEL_COURSE_RUN: "2018"
 WHITELABEL_MODULE_STORE: "split"
+
+
+CREATE_SITES_AND_CONFIGURATIONS_COMMAND: "create_sites_and_configurations"
```
Comment (thraxil) [inline playbooks/roles/whitelabel/defaults/main.yml:9]:
```suggestion
CREATE_SITES_AND_CONFIGURATIONS_COMMAND: "create_sites_and_configurations"
```

Comment (thraxil) [reply playbooks/roles/whitelabel/defaults/main.yml:9]: (adding a newline)

**Review by thraxil — COMMENTED** [review]
AFAICT, none of our playbooks include the `whitelabel` role directly or indirectly. If we want this to actually do anything, it will need to be added somewhere. But doing that, we'd want to review the entire role to make sure that the other things it does (eg, creating courses, sites, and partners) are things that we want to happen.

**Comment by johnbaldwin** [comment]
Thanks @thraxil . Given what you typed above, I'm just going to close this PR w/out merging. cc @OmarIthawi

**Pull request closed without merging**